src/lib/journey/callbacks/kba/kba-create.svelte

```diff
@@ -57,6 +60,12 @@
      *********************************************************************** */
     callback.setQuestion((event.target as HTMLSelectElement).value);
   }
+
+  $: {
+    if (callback) {
+      value = '';
```

Comment:
Curious, why are we doing this just for KBA create and not any of the other callbacks?

Reply:
Good catch, this is no longer needed. I will remove with my update later today.